src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_data_containers.h

```diff
@@ -827,7 +1062,7 @@ namespace sirf {
 		{
 			return new AcquisitionsVector(info, ignore_mask_);
 		}
-		virtual ObjectHandle<DataContainer>* new_data_container_handle() const
+		virtual ObjectHandle<DataContainer>* new_data_container_handle(const bool initialise_with_0) const
```

Comment:
will have to fill with 0 if `initialise_with_0=true`

Reply:
done